tests/foreman/virtwho/cli/test_esx.py

```diff
@@ -473,3 +473,48 @@ def test_positive_deploy_configure_hypervisor_password_with_special_characters(
         )
         target_sat.cli.VirtWhoConfig.delete({'name': virtwho_config['name']})
         assert not target_sat.cli.VirtWhoConfig.exists(search=('name', form_data['name']))
+
+    @pytest.mark.tier2
+    def test_positive_remove_env_option(self, default_org, form_data, virtwho_config, target_sat):
+        """remove option 'env=' from the virt-who configuration file and without any error
+
+        :id: 509add77-dce7-4ba4-b9e5-2a5818c39731
+
+        :expectedresults:
+            the option "env=" should be removed from etc/virt-who.d/virt-who.conf
+            /var/log/messages should not display warning message
+
+        :CaseLevel: Integration
+
+        :customerscenario: true
+
+        :CaseImportance: Medium
+
+        :BZ: 1834897
+
+        :customerscenario: true
```

Comment:
Same here

Reply:
> Same here
Have remove the duplicated :customerscenario: true